### PR TITLE
Write release notes for Chef 13

### PR DESCRIPTION
**Pull request by thommay**
First pass, review and updates appreciated @chef/maintainers.

I used https://github.com/search?l=&o=asc&p=1&q=type%3Apr+merged%3A%222017-02-23+..+%2A%22+repo%3Achef%2Fchef&ref=advsearch&s=updated&type=Issues&utf8=%E2%9C%93 as a query for all the PRs we merged since 12.19.36 .

### CODE REVIEW

**Comment by thommay** [comment]
@smurawski yep, feel free to push more commits to this :)

File: Gemfile [hunk 1]
```diff
@@ -12,7 +12,7 @@ source "https://rubygems.org"
 gem "chef", path: "."
 
 # tracking master of ohai for chef-13.0 development, this should be able to be deleted after release
```
Comment (lamont-granquist) [inline Gemfile]: can delete the comment too

File: Gemfile.lock [hunk 2]
```diff
@@ -1,6 +1,6 @@
 GIT
   remote: https://github.com/chef/chef-server
-  revision: 07fea253ceea254af8fbd9c602a04017751cae21
+  revision: dfebe05bf65f4d55483f7be80971cb0b75da74c8
```
Comment (lamont-granquist) [inline Gemfile.lock]: this made travis red the last time i tried it

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -2,7 +2,101 @@ _This file holds "in progress" release notes for the current release under devel
...
+
+This is the inverse of the pre-existing whitelisting functionality.
+
+## The guard interpreter for `powershell_script` is Powershell, again
+
+When writing `not_if` or `only_if` statements, by default we now run
+those statements using powershell, rather than forcing the user to set
+`guard_interpreter` each time.
+
+## Zypper GPG checks by default
```
Comment (lamont-granquist) [inline RELEASE_NOTES.md]: this might be backcompat breaking

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -2,7 +2,101 @@ _This file holds "in progress" release notes for the current release under devel
...
+In the case where a resource must force the use of only the specified source(s), then the `include_default_source` property
+has been added -- setting it to false will remove the `Chef::Config[:rubygems_url]` setting from the list of sources for
+that resource.
+
+The behavior of the `clear_sources` property is now to only add `--clear-sources` and has no magic side effects on the source options.
+
+## Ruby version upgraded to 2.4.1
+
+We've upgraded to the latest stable release of the Ruby programming
+language.
```
Comment (danielsdeleo) [inline RELEASE_NOTES.md]: Link to Ruby's release notes would be a nice bonus here.

File: RELEASE_NOTES.md (hunk 3 above)
Comment (thommay) [inline RELEASE_NOTES.md]: i think it's a feature and we should market it as such :)

Comment (lamont-granquist) [inline RELEASE_NOTES.md]: 'k

Comment (danielsdeleo) [inline RELEASE_NOTES.md]: Compat is the reason we waited to do it, but I'm not opposed to that reasoning either.

**Pull request merged**